ts/src/bitget.ts

```diff
@@ -2333,14 +2333,14 @@ export default class bitget extends Exchange {
      */
     async withdraw (code: string, amount: number, address: string, tag = undefined, params = {}): Promise<Transaction> {
         this.checkAddress (address);
-        const chain = this.safeString2 (params, 'chain', 'network');
-        params = this.omit (params, 'network');
```

Comment:
I think we were not doing correctly, if user provides `chain: xyz` then it is exchange specific param and we should not hook into it to parse it through `networkCodeToId` several lines below.
we should `handleNetworkCodeAndParams` and then `networkCodeToId` for unified `network` param key, but should not touch user-provided exchange-specific param (e.g. `chainId` or whatever)  value.